src/components/map/MapComp.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Map } from '@flybywiresim/map';
+
+export const MapComp: React.FC = () => {
+    return (
+        <>
+            <div className=" h-screen bg-blue-dark-contrast">
+                <div className="relative h-screen pt-20 text-black z-20">
+                    <Map refreshInterval={10_000} disableMenu={true} disableWeather={false} disableScroll={false} forceTileset={'carto-light'} />
+                </div>
+            </div>
```

Comment:
The map has most of its features disabled on the home page. By removing all properties these features get unlocked which we should definitely do on the fullscreen version.

```suggestion
            <div className=" h-screen bg-blue-dark-contrast">
                <div className="relative h-screen pt-20 text-black z-20">
                    <Map />
                </div>
            </div>
```

Please also take a look at the community section. @Benjozork implemented logic to throttle updates when the map isn't in the foreground. This throttling should be used here as well.